src/components/controllers/page/index.js

```diff
@@ -158,7 +158,7 @@ export default function Page({ route, prev, next }, ctx) {
 	// "current" is the currently *displayed* page ID.
 
 	const showTitle = current != 'index' && meta.show_title !== false;
-	const canEdit = showTitle && current != '404' && current !== '/blog';
```

Comment:
@marvinhagemeister Looks like this was done when you created the blog originally, do you remember the reasoning behind it?

Currently it doesn't work, as the page is just `blog`, without the leading `/`, but I don't necessarily see any reason why it should not be editable; in fact, someone's been kind enough already to add a [Chinese translation](https://github.com/preactjs/preact-www/blob/master/content/zh/blog.md).

Reply:
@rschristian Apologies for seeing the mention so late. I think originally when prototyping the blog we had nothing that could be translated on that page. Might have changed in a newer iteration so I likely just forgot to update this part.

Reply:
No worries!

And fair enough, just wanted to check to make sure I wasn't overlooking anything.